docker.d/init_worker.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 set -e
 
+export JARSIGNER_KEY_STORE="/app/mozilla-android-keystore"
+rm -f "$JARSIGNER_KEY_STORE"
+JARSIGNER_KEY_STORE_PASSWORD=$(tr -cd '[:alnum:]' < /dev/urandom | fold -w30 | head -n1)
```

Comment:
Would it be possible to throw a comment here explaining what this does, for the non-wizards in the audience?

Reply:
Good point, I'll add.